java/src/test/java/testLinkedList/testLinkedList.java

```diff
@@ -0,0 +1,95 @@
+package testLinkedList;
+
+import LinkedList.LinkedList;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+public class testLinkedList {
+
+    @Test
+    public void testInstantiate() {
+        LinkedList empty = new LinkedList();
+        int expected = 0;
+        int actual = empty.getSize();
+        assertEquals("Linked list length =/= expected", expected, actual);
+    }
+
+    @Test
+    public void testInsertion() {
+        LinkedList oneInsertion = new LinkedList();
+        oneInsertion.addToStart(7);
+        //True
+        assertEquals(7, oneInsertion.find(7).getValue());
+        assertTrue(1 == oneInsertion.getSize());
+        //False
+        assertEquals(null, oneInsertion.find(14));
+        assertFalse(1 != oneInsertion.getSize());
+    }
+
+    @Test
+    public void testHead() {
+        LinkedList header = new LinkedList();
+        header.addToStart(1);
+        header.addToStart(2);
+        header.addToStart(3);
+        header.addToStart(4);
+        header.addToStart(5);
+
+        assertEquals(5, header.getHead());
+    }
+
+    @Test
+    public void testMultipleInsertions() {
+        LinkedList multipleInsertions = new LinkedList();

```

Comment:
Include comments in your tests, or messages in your assert statements for what you are expecting the test to do.